src/Sylius/Bundle/CoreBundle/Fixture/Factory/ChannelExampleFactory.php

```diff
@@ -123,6 +133,9 @@ protected function configureOptions(OptionsResolver $resolver)
                 return $this->faker->boolean(90);
             })
             ->setAllowedTypes('enabled', 'bool')
+            ->setDefault('default_tax_zone', LazyOption::randomOneOrNull($this->zoneRepository, 100))
```

Comment:
`LazyOption::randomOneOrNull($this->zoneRepository, 100)` is equivalent to `LazyOption::randomOne($this->zoneRepository)`